pennylane/devices/tests/test_gates.py

```diff
@@ -421,6 +421,8 @@ def test_two_qubit_no_parameters(self, device, init_state, op, mat, tol, skip_if
         n_wires = 2
         dev = device(n_wires)
         skip_if(dev, {"returns_probs": False})
+        if not dev.supports_operation(op):
```

Comment:
@albi3ro How come the `skip_unsupported` mark didn't work here? :thinking: I'd think that helps skip the unsupported operations.

Reply:
Well... this works and is less confusing? I think to use the marker to skip tests, I would have to pass in a command line option to pytest?

Reply:
Just adding a quick comment here, but I believe the original design around the marker was deliberate.

This is so that, if you are building a plugin and want to ensure it supports _all_ PL ops, you simply leave off `--skip-ops` (or whatever the CLI flag is), and you will get failures for any unsupported operations